main.py

```diff
@@ -107,6 +121,74 @@ def get_name_tag(resource_type, name):
 )
 
 
+# Initialize the ELBv2 (ALB) client
```

Comment:
Ideally all the code below should be added separately in #19.

Let's revert the changes from below and address them in that separate pull request.